minijson.h

```diff
@@ -429,7 +449,30 @@ std::string CodePointToUTF8(unsigned long int code) {
             static_cast<char>(0x80 | ((code >> 6) & 0x3f)),
             static_cast<char>(0x80 | (code & 0x3f))};
   }
-  throw std::runtime_error("Code point out of normal people range.");
+  throw std::runtime_error("Code point out of usual range.");
+}
+
+// https://linux.die.net/man/7/utf8
+unsigned long int
+UTF8ToCodePoint(BoundIterator<std::string::const_iterator> *it) {
```

Comment:
From [rfc7159](https://tools.ietf.org/html/rfc7159#section-7):
"All Unicode characters **may** be placed within the
   quotation marks, except for the characters that must be escaped:
   quotation mark, reverse solidus, and the control characters (U+0000
   through U+001F).

Any character **may** be escaped [...]"

I think we can just spit out utf-8 strings here, no need to escape-encode them.

Reply:
Yes good point thanks